documentation/custom-transformers.md

```diff
@@ -5,17 +5,18 @@ purpose can be leveraged.
 Individual custom transformers can be composed together from any number of external repositories and executed as a
 single process using the `compose` and `execute` commands or the `composeAndExecute` command. This is accomplished by
 generating a Go plugin which allows the `vulcanizedb` binary to link to the external transformers, so long as they
-abide by one of the standard [interfaces](../staging/libraries/shared/transformer).
+abide by the [event transformer interface](../libraries/shared/factories/event/transformer.go) or the
+[storage transformer interface](../libraries/shared/factories/storage/transformer.go).
 
 ## Writing custom transformers
 For help with writing different types of custom transformers please see below:
 
 Storage Transformers: transform data derived from contract storage tries
-   * [Guide](../../staging/libraries/shared/factories/storage/README.md)
-   * [Example](../../staging/libraries/shared/factories/storage/EXAMPLE.md)
+   * [Guide](../libraries/shared/factories/storage/README.md)
+   * [Example](../libraries/shared/factories/storage/EXAMPLE.md)
```

Comment:
What's up with this diff? Were these broken previously, or does this change what branch we look at?

Reply:
I noticed that these links weren't working - not sure entirely how `staging` in the link was affecting things, but I noticed when I removed it, the link would still redirect to staging. 🤔